model/AI/Vocabularies/ModalityType.md

```diff
@@ -0,0 +1,30 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# ModalityType
+
+## Summary
+
+Specifies the modality of the inputs or outputs used by different Foundation Model (FM) based software components, like prompts and agents. 
+
+## Description
+
+List the different acceptable modalities of input or output used by the different FM-based components.
+
+If the modality used by your component is not listed here, please select "other".
+
+## Metadata
+
+- name: ModalityType
+
+## Entries
+
+- 3D: Point clouds or 3D objects.
+- audio: Speech or sound.
+- geospatial: Maps or location data.
+- image: Static visual content.
+- multimodal: Combination of two or more modalities.
+- Tabular: Row–column structured data.
+- text: Written language.
+- Time-series: Data over time.
```

Comment:
```suggestion
- timeseries: Data over time.
```

The parser failed here because of `-` in the entry name.
Entry name can't have space or punctuations.
It uses lower camel case (first word starts with lower case).

So it should be `timeSeries` -- but since [DatasetType](https://github.com/spdx/spdx-3-model/blob/develop/model/Dataset/Vocabularies/DatasetType.md) already use `timeseries`, we may like to use the same casing.